WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/MapViewHolder.kt

```diff
@@ -24,10 +24,11 @@ class MapViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
         parent,
         R.layout.stats_block_web_view_item
 ) {
+    private val coroutineScope = CoroutineScope(Dispatchers.Default)
     private val webView: WebView = itemView.findViewById(R.id.web_view)
     @SuppressLint("SetJavaScriptEnabled")
     fun bind(item: MapItem) {
-        GlobalScope.launch {
+        coroutineScope.launch {
```

Comment:
I think this was wrong before. I'm pretty sure this previously ran on the main thread and jumped on the main thread later. I don't think that makes sense (and it wasn't the original idea).